src/yuzu/install_dialog.h

```diff
@@ -0,0 +1,33 @@
+// Copyright 2020 yuzu Emulator Project
+// Licensed under GPLv2 or any later version
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <QCheckBox>
+#include <QDialog>
+#include <QDialogButtonBox>
+#include <QLabel>
+#include <QLayout>
+#include <QListWidget>
+
+class InstallDialog : public QDialog {
+    Q_OBJECT
+
+public:
+    explicit InstallDialog(QWidget* parent, const QStringList& files);
+    ~InstallDialog() override;
+
+    QStringList GetFilenames() const;
+    bool OverwriteFiles() const;
```

Comment:
```suggestion
    bool ShouldOverwriteFiles() const;
```

Since this is only checking if the files should be overwritten, it isn't actually overwriting them.